scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+    above the WGS84 ellipsoid. This coordinate will be communicated to the Rover
+    against which to calculate a pseudo-absolute position. This value should be as
+    accurate as possible and should be precise to 1 cm.  Any errors in the surveyed
+    position will directly affect the pseudo-absolute RTK position measurement reported
+    by the Rover.
+- group: float_kf
+  name: phase_var
+  type: Double
+  units: cycles^2
+  default value: '0.0144'
+  enumerated possible values:
+  Description: Assumed variance of a satellite's phase measurement
+  Notes: This setting adjusts variance estimates in the Swift Kalman filter which
+    aids in integer ambiguity resolution (IAR).  Increasing this value can reduce
+    the occurrence of false carrier phase locks but can also increase the time required
+    to achieve an IAR fixed solution.  This setting should not be adjusted by end
+    users.
+- group: float_kf
+  name: code_var
+  type: Double
+  units: meters^2
+  default value: '40000'
+  enumerated possible values:
+  Description: Assumed variance of a satellite's pseudorange measurement
+  Notes: This setting adjusts variance estimates in the Swift Kalman filter which
+    aids in integer ambiguity resolution (IAR).  Increasing this value can reduce
+    the occurrence of false carrier phase locks but can also increase the time required
+    to achieve an IAR fixed solution.  This setting should not be adjusted by end
+    users.
+- group: float_kf
+  name: amb_init_var
+  type: Double
+  units: non dimensional
+  default value: '1.00E+08'
+  enumerated possible values:
+  Description: Initial integer ambiguity variance at filter initialization
+  Notes: This setting adjusts variance estimates in the Swift Kalman filter which
+    aids in integer ambiguity resolution (IAR).  Increasing this value can reduce
+    the occurrence of false carrier phase locks but can also increase the time required
+    to achieve an IAR fixed solution.  This setting should not be adjusted by end
+    users.
+- group: float_kf
+  name: new_amb_var
+  type: Double
+  units: non dimensional
+  default value: '1.00E+10'
+  enumerated possible values:
+  Description: Variance for new ambiguity measurements
+  Notes: This setting adjusts variance estimates in the Swift Kalman filter which
+    aids in integer ambiguity resolution (IAR).  Increasing this value can reduce
+    the occurrence of false carrier phase locks but can also increase the time required
+    to achieve an IAR fixed solution.  This setting should not be adjusted by end
+    users.
+- group: frontend
+  name: antenna_selection
+  type: enum
+  units:
+  default value: Auto
+  enumerated possible values: Auto,Patch,External
+  Description: Determines which antenna to use.
+  Notes: This setting selects the antenna input that should be used by the Piksi.  When
+    set to auto, if the unit senses an external antenna attached, it will use the
```

Comment:
Maybe mention that it senses an external active antenna is present by its bias current draw? Could catch someone out as if they connected an external passive antenna that would not be detected.